x-pack/plugins/observability_solution/observability_onboarding/public/assets/auto_detect.sh

```diff
@@ -382,7 +371,7 @@ known_integration_title() {
       echo "Docker Container Logs"
       ;;
     "system")
-      echo "System Logs"
+      echo "System Logs And Metrics"
```

Comment:
I'm struggling a bit with the titles.

We mention a bit too explicitly whether we detected logs or metrics when in reality we don't know at this stage.

For example we could have detected metrics only without any logs for the system integration. Same for the docker integration (I've added a [fix](https://github.com/elastic/kibana/pull/189335#discussion_r1694297943) to detect docker metrics when no docker logs are available but that kind of doesn't work with the title "Docker Logs" we use)

Could we either be a bit more vague about what we detected or alternative tie in the detection logic with the title so that we construct the correct title based on what was detected.

Reply:
Yeah, agree, I'd also prefer to be a bit more vague here something like "System Logs And/Or Metrics" but actually usable. Could not come up with anything good... Also, did not want to complicate the logic with a dynamic title, but maybe you're right and there is no way around it.

Reply:
I don't have a better idea either but agree that creating a dynamic title is overkill. 

Let's leave it as is for now then